data/projects/improve_installation.md

```diff
@@ -18,30 +18,72 @@ Each bear has a REQUIREMENTS tuple which holds instances of requirement classes,
 found here (https://gitlab.com/coala/package_manager), which should be
 reworked so that most operations in cib would be methods from these classes.
 
-As a start for this project, EACH bear must have all its dependencies solved
-out (>80% do), including the platform dependent ones. For this to be done,
-DistributionRequirement should be reworked.
+The problem right now is that some bears are not even getting installed by
+cib. Besides those with which we have requirement classes, the rest
+aren't even treated. So the plan is to create package managers for the
+popular managers, and treat them all in an universal DistributionRequirement.
+
+How this will work: Each requirement class will have an "install_command" and
+an "install" method. We will just call requirement.install() and this will
+run the command on its' own, not caring which platform you're on. Ideally,
+requirement.install() will use "install_command" to generate the command and
+run it with "sarge". For the DistributionRequirement class, the
+"install_command" method will check on which platform you are, and it will
+generate the according command.
```

Comment:
Could this be `generate the respective command` 
Or `generate the command accordingly`